Datasnap/Classes/Event Properties/Identifier.m

```diff
@@ -46,8 +47,25 @@ - (NSDictionary*)convertToDictionary
         @"mobile_device_google_advertising_id" : self.mobileDeviceGoogleAdvertisingId ? self.mobileDeviceGoogleAdvertisingId : [NSNull null],
         @"mobile_device_google_advertising_id_opt_in" : self.mobileDeviceGoogleAdvertisingIdOptIn ? self.mobileDeviceGoogleAdvertisingIdOptIn : [NSNull null],
         @"unknown" : self.unknown ? self.unknown : [NSNull null],
-        @"facebook_uuid" : self.facebookUuid ? self.facebookUuid : [NSNull null]
+        @"facebook_uuid" : self.facebookUuid ? self.facebookUuid : [NSNull null],
+        @"hashed_email" : [self sha1:self.hashedEmail]
```

Comment:
Why is self.hashedEmail being converted to sha1 here?  With the property name hashedEmail I would assume it would already be converted if I wanted to access it somewhere else within the code